examples/plugins/hello-world/Makefile

```
@@ -0,0 +1,93 @@
+.PHONY: build clean install help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)